scripts/k8s.env

```diff
@@ -72,14 +72,16 @@ ETCD_VERSION=3.5.16
 #   template creation process.
 
 ### Debian 12 Image (Bookworm)
+TEMPLATE_VM_BRIDGE=vmbr0
 TEMPLATE_VM_ID=9000
 TEMPLATE_VM_NAME="k8s-ready-template"
 IMAGE_NAME="debian-12-generic-amd64.qcow2"
 IMAGE_LINK="https://cloud.debian.org/images/cloud/bookworm/latest/${IMAGE_NAME}"
 EXTRA_TEMPLATE_TAGS="bookworm template"
-TEMPLATE_DISK_SIZE=4.3G # Add 1.6G more if installing nvidia drivers
+TEMPLATE_DISK_SIZE=4.6G # Add 1.6G more if installing nvidia drivers
 
 ### Ubuntu 24.04 LTS Image (Noble Numbat)
+#TEMPLATE_VM_BRIDGE=vmbr0
```

Comment:
When the other line is moved you can remove this

Reply:
Done